tests/plugins/test_pull_base_image.py

```diff
@@ -101,7 +115,7 @@ def test_pull_base_image_plugin(parent_registry, df_base, expected, not_expected
     workflow.builder.base_image = ImageName.parse(df_base)
 
     expected = set(expected)
-    expected.add(df_base)
```

Comment:
`df_base` doesn't have the registry name included, which is why it was missing. Pulling e.g. library/foo results in example.registry.com/library/foo being present.